packs/bestiary-effects/effect-grasping-tendrils.json

```diff
@@ -20,7 +20,17 @@
             "remaster": false,
             "title": "Pathfinder Society Scenario #5-09: Equal Exchange - Skymetal Hoard"
         },
-        "rules": [],
+        "rules": [
```

Comment:
I think the REs for this are already on the creatures - the effect merely exists to predicate on.

Reply:
Yep, I see that now. Fixed.